### PR TITLE
Tips - "Double Tab completion" and "Drag folder or file to get path 

### PR DESCRIPTION
**Pull request by ruudmens**
### Summary

Add a tip to set configure pressing tab twice to view all parameters, properties and possibilities.

### Checklist

-

- [ ] Tests have been added for this code change (if applicable)
- [ ] Docs have been added or updated (if applicable)
- [X] Code format follows the project style
- [ ] All new and existing tests passed

### What type of changes does this PR include

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Describe the change in more detail

-

### Additional information

-

### CODE REVIEW

File: src/PowerShellTips/2024-09-23-press-tab-twice-to-view-all-parameters-properties-and-possibilities.ps1 [hunk 1]
```diff
@@ -0,0 +1,34 @@
+$tip = [tiPS.PowerShellTip]::new()
+$tip.CreatedDate = [DateTime]::Parse('2024-09-23')
+$tip.Title = 'Press Tab twice to view all parameters, properties and possibilities'
+$tip.TipText = @'
+Add `Set-PSReadlineKeyHandler -Key Tab -Function Complete` to your PowerShell profile. This way you can press the Tab key twice after a command to view all possible parameters, properties, and possiblities.
```
Comment (deadlydog) [inline src/PowerShellTips/2024-09-23-press-tab-twice-to-view-all-parameters-properties-and-possibilities.ps1:5]:
```suggestion
Add `Set-PSReadlineKeyHandler -Key Tab -Function Complete` to your PowerShell profile. This way you can press the Tab key twice after a command to view all possible parameters, properties, and possibilities.
```

Spellcheck is complaining about this typo

**Comment by deadlydog** [comment]
Great tips. Thanks for contributing! 😀

File: src/PowerShellTips/2024-09-23-press-tab-twice-to-view-all-parameters-properties-and-possibilities.ps1 (hunk 1 above)
[the remark this answers is not in the document]
Comment (deadlydog) [reply src/PowerShellTips/2024-09-23-press-tab-twice-to-view-all-parameters-properties-and-possibilities.ps1:5]: Also, now that I've approved a workflow run for you, I believe the workflows should run right away for any other PRs you submit, allowing you to get faster feedback about pipeline problems like this one.

[the remark this answers is not in the document]
Comment (ruudmens) [reply src/PowerShellTips/2024-09-23-press-tab-twice-to-view-all-parameters-properties-and-possibilities.ps1]: I have fixed the typos (and enabled the spell check extension in VS), so should be all good now.

**Pull request merged**